pkg/api/ldap_debug.go

```diff
@@ -0,0 +1,119 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/ldap"
+	"github.com/grafana/grafana/pkg/services/multildap"
+)
+
+var (
+	getLDAPConfig = multildap.GetConfig
+	newLDAP       = multildap.New
+)
+
+// LDAPAttribute is a serializer for user attributes mapped from LDAP. Is meant to display both the serialized value and the LDAP key we received it from.
+type LDAPAttribute struct {
+	ConfigAttributeValue string `json:"cfgAttrValue"`
+	LDAPAttributeValue   string `json:"ldapValue"`
+}
+
+// RoleDTO is a serializer for mapped roles from LDAP
+type RoleDTO struct {
+	OrgId   int64           `json:"orgId"`
+	OrgRole models.RoleType `json:"orgRole"`
+	GroupDN string          `json:"groupDN"`
+}
+
+// TeamDTO is a serializer for mapped Teams from LDAP
+type TeamDTO struct {
+	GroupDN string `json:"groupDN"`
+	TeamId  int64  `json:"teamId"`
```

Comment:
team name?

Reply:
Interesting. I took inspiration from our current users API  (`/api/users`) where we only expose the `OrgId` - I assumed this is for issuing a separate request on the front-end for each organisation.

Is that not the case? I'm happy to change the structure and make the queries to the db right here but not sure that's what we do in other areas.

Reply:
Not sure I understand you. Without the team name how can we show this in the table in the UI?

Reply:
We had a clear-up on Slack about this one. The summary of the discussion is: 

> Generally, for list APIs we usually try to include everything needed by UI